tests/utils.go

```diff
@@ -243,6 +243,9 @@ type ObjectEventWatcher struct {
 	dontFailOnMissingEvent bool
 }
 
+const PromptExpression = "\\$ |\\# "
```

Comment:
as spoken on other context, maybe good to add here ( ), so it wont connect to the stuff before it, and the | would be wrongly attached ?

Reply:
As I wrote on the other context. I found this bug here on this PR so of course it will be done (already done two minutes after I wrote the previous comment) on this PR as well.

Reply:
> as spoken on other context, maybe good to add here ( ), so it wont connect to the stuff before it, and the | would be wrongly attached ?

It is extremely odd to see this commented here when the author of this PR commented this already on https://github.com/kubevirt/kubevirt/pull/3883/files#r467720987.

Reply:
the original message was edited there, so i looked on the 2nd argument in the code, and then it was edited to fit that because this was the original context.

Reply:
The other field doesn't even contain "#" and doesn't need parentheses, so it is pretty clean it was a typo.
Anyway, never mind. Thanks for caring:)

Reply:
its was pure intentions, my mind when you talked about X ,looked on Y (which is the 2nd param),
as i wrote above "and then it was edited to fit that because this was the original context."

Reply:
ok, thanks.